src/bob/resource/internals.clj

```diff
@@ -142,6 +142,18 @@
   (some? (db/invalid-external-resources states/db
                                         {:name (:provider resource)})))
 
+(defn get-resource-params
+  "Fetches list of parameters associated with the resource"
+  [pipeline name]
+  (reduce
```

Comment:
can use https://clojuredocs.org/clojure.core/into here

Reply:
Can you help me with this one, I don't quite understand it.

Reply:
As far as I understand, you're trying to create a single map from a list of maps
right? This can be done using into like this:
```clojure
(into {} (db/resource-params-of ...))
```
this can be returned directly.
correct me if I'm understaning it wrong.

Reply:
ah yes,i forgot about the other keys. its okay then. could you add an unit test for this
function?